sites/fast-website/src/app/css/style.css

```diff
@@ -31,34 +31,23 @@ body {
   z-index: 1;
 }
 
-.social-links {
-  position: fixed;
-  top: 45%;
-}
-
-.section-scroll {
+.side-nav-container {
+  width: 100%;
+  height: min-content;
   position: fixed;
-  top: 40%;
-  right: 0%;
+  top: 50%;
+  transform: translateY(-50%);
+  display: flex;
+  justify-content: space-between;
+  z-index: -1;
```

Comment:
Do we need z-index here? Why do we need to bury this container?

Reply:
Without the `z-index`, the container sits on top of the content and prevents interacting with any content in the center of the page.